test/non-init.test.js

```diff
@@ -0,0 +1,523 @@
+
+const path = require("path")
+// const BUCKET = process.env.BUCKET
```

Comment:
i dont think this needs all of these test cases and a seperate file 

just need one test case to test that the same client is returned both ways